support-frontend/assets/pages/[countryGroupId]/components/checkoutComponent.tsx

```diff
@@ -617,8 +617,13 @@ export function CheckoutComponent({
 					`An error occurred in checkoutComponent.tsx while trying to submit the form: ${errorMessage}`,
 				);
 			}
+			// This state update is in the catch block because it has the effect of
```

Comment:
The description is perfect, do you need previously in the comments if its in the PR though?